### ISSUE TITLE
System.IO.FileNotFoundException: Could not load assembly 'MathParser.org-mXparser.Stand16. Perhaps it doesn't exist in the Mono for Android profile?

### ISSUE THREAD

**Issue by harindaka** [mariuszgromada/MathParser.org-mXparser#141]
### Issue title:
- Why is the .Net Runtime unable to find the MxParser Assembly in a Xamarin Project?

### Issue content:
I added the MathParser.org-mXparser nuget package v4.1.1 to a Xamarin Forms project. I installed the nuget package to the shared project (.net standard 2.0) and Xamain.Android project as well. Package installation went well. However when I try to run the Android project I get the following error:

`
Exception while loading assemblies: System.IO.FileNotFoundException: Could not load assembly 'MathParser.org-mXparser.Stand16, Version=4.0.0.2, Culture=neutral, PublicKeyToken=0fbab7add73ed6e2'. Perhaps it doesn't exist in the Mono for Android profile?
`

- MathParser.org-mXparser verssion: v.4.1.1
- Framework: .net standard 2, Xamarin, Xamain.Android, Xamain.iOS

I later downgraded the library to .netstandard 1.6 but I get the same error. How can I solve this?

**Comment by harindaka** [comment]
I was able to get it working with the following steps

1. Download the source code for mxparser. 
2. Add only the project "mXparser-41-dot.stand.1.6" from the mxparser source code to my Xamarin solution
3. Updated the .Net Standard setting in the mXparser-41-dot.stand.1.6 project settings from .netstandard1.6 to 2.0
4. Reference the mXparser-41-dot.stand.1.6 project from my Xamarin Forms project
5. Run the Xamain.Android project.

1. Why would this work but not the nuget package? 
2. What could go wrong if I continue to use the mXparser-41-dot.stand.1.6 library this way?
3. If it is really this easy to upgrade mXparser-41-dot.stand.1.6 to .netstandard 2.0 why don't we have .netstandard2.0 support yet?
